dist/test_data.py

```diff
@@ -215,7 +215,9 @@ def __ge__(self, other):
 
 methods = {
     'base_test' : Method(test_config),
-    'burst_inserts' : Method(test_config),
+    'burst_inserts' : Method(test_config + [
+                        Config("burst_duration", 90, r'''
```

Comment:
This is a great example of how we can add a specific field to a configuration, nice!